### PR TITLE
TableNG: BarGauge cell updates

### LINKED ISSUE #94769: Table: Cell Type - Confirm Parity for Gauge
Gauge is already in the [Table POC](https://github.com/grafana/grafana/tree/table-ng). This task is to confirm complete feature parity:
Gauge display mode: basic, gradient, retro LCD
Value display: value color, text color, hidden

Image

### PR DESCRIPTION
**Pull request by adela-almasan**
Before

https://github.com/user-attachments/assets/e3e92350-6d69-4b5f-9002-dff5c06e28ba

After

https://github.com/user-attachments/assets/10895187-4d3d-446d-ab50-f84ebc557b68

Fixes https://github.com/grafana/grafana/issues/94769

**Special notes for your reviewer:**
- we need to fix the style, in the case of the gauge cell the numbers don't show up. I'll create a [follow up](https://github.com/grafana/grafana/issues/95570)

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx [hunk 1]
```diff
@@ -145,10 +145,11 @@ export function TableNG(props: TableNGProps) {
             <TableCellNG
               key={key}
               value={value}
-              field={shallowField}
+              field={field}
               theme={theme}
               timeRange={timeRange}
-              height={rowHeight}
+              height={rowHeight()}
+              {...props}
```
Comment (Develer) [inline packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx:152]: Let's better specify props, but we of course can do it later.

**Comment by Develer** [comment]
It took like 8 seconds to switch to `Retro LCD` mode 🤔
https://github.com/user-attachments/assets/cdeded89-5b0b-4ef8-a99f-bbaf10795502

**Comment by Develer** [comment]
It took like 8 seconds to switch to Retro LCD mode 🤔


https://github.com/user-attachments/assets/8510d430-4105-460c-ac9f-b68fe87af718

**Comment by Develer** [comment]
Also noticed blinking, which is stop if resizing happening in small width 🤷🏻‍♂️


https://github.com/user-attachments/assets/53f59aea-469f-4250-bcec-866736692249

**Comment by adela-almasan** [comment]
> Also noticed blinking, which is stop if resizing happening in small width 🤷🏻‍♂️
> 
>  Screen.Recording.2024-11-01.at.18.16.35.mov

I think this one is related to the width. I opened an issue for it [here](https://github.com/grafana/grafana/issues/95570).

**Comment by adela-almasan** [comment]
Hm, can't repro the the issue when switching between the display mode 🤔 My guess is that the overall performance for these cells will improve when we fix the inner width. I'll try to look into that this week.

**Review by drew08t — APPROVED** [review]
LGTM and works as expected with the following documented limitations for follow up:

1. Issues with smaller column widths
2. Actions

**Pull request merged**